validateApiDefinition.js

```diff
@@ -68,146 +68,116 @@ export const validateDefinition = async (apiDefinition, fileName, validationLeve
     // Version field is not validated if the definition is not a valid json
   }
 
-  if (validationLevel === 1) {
-    return spectral.run(myDocument).then(async (result) => {
-      let level1WarnList = [];
```

Comment:
Do we no longer give level 1 support? If that is the case, we can remove all the above helper texts that mention that there are two validation levels; the readme and index.js

Reply:
since we are providing APIM 4.0.0 behavior in level 1, still we need to supress some of the errors. isn't it ?

Reply:
Are we giving a level 1 support with the node tool? If yes, this should be fine

Reply:
yes, we are